randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -286,8 +286,11 @@ Extra - room_id: [9]
   * L3 Hatch to Security Save Room/Door to Security Shaft West
   * Extra - door_idx: (18,)
   > Door to Level 3 Security Room
-      # TODO: varia / damage reqs
-      Varia Suit
+      All of the following:
+          # TODO: dmg run
+          Varia Suit
+          # Get past enemies
+          Normal Damage ≥ 40 or Can Kill Tough Beam-Weak Enemy
```

Comment:
which enemies? I only see the blue-x and the crabs, but surely the crabs arent that difficult that they actually neeed reqs?

Reply:
lmao



they need it

Reply:
would change the comment to "Get past <insert crab name here>"

Reply:
oh  i just realized github hid the meaning whoops.
meant to write "Get past \< insert crab name here \>"